pyqir-parser/tests/test_api.py

```diff
@@ -68,6 +68,19 @@ def test_parser_select_support():
     assert isinstance(instr2.false_value, QirLocalOperand)
     assert instr2.false_value.name == "val.i.1"
 
+def test_global_string():
```

Comment:
@swernli  This sample shows reading ascii data converted into utf-8. Does it make sense to have a unicode byte array sample/test?

Reply:
Right now we don't have any tools that produce unicode byte arrays, so we'd have to try and author one manually. I'm not sure how to do that at the moment, so this is the only test case we have.